File: HRO/CMITDW Taal der Wiskunde/o22.java
// opdracht 2.2; Productsymbool
public class o22
{
	public static void main(String[] args)
	{
		System.out.println(product(1, 99));
	}
	
	public static double product(int a, int b)
	{
		double sum = 1;
		for (int i=a; i<=b; i++)
		{
			sum *= (double) i / (i + 1);
			System.out.println(i+"\t"+sum);
		}
		return sum;
	}
}

File: HRO/CMITDW Taal der Wiskunde/Lua/o22.lua
-- opdracht 2.2; Productsymbool
local function product(a, b)
	local sum = 1
	for i = a, b do
		sum = sum * (i / (i + 1))
		print(i, sum)
	end
	return sum
end

print(product(1, 99))
